backend/src/services/model.ts

```diff
@@ -222,6 +230,11 @@ export async function _setModelCard(
   }
 
   const revision = new ModelCardRevisionModel({ ...newDocument, modelId, createdBy: user.dn })
+  const auth = await authorisation.model(user, model, ModelAction.Write)
```

Comment:
No need to update authorisation here as we're no longer using it to check mirrored models.